src/praisonai-bot/praisonai_bot/bots/_registry.py

```
@@ -90,6 +90,138 @@ def __init__(self):
+        for _, obj in inspect.getmembers(module, inspect.isclass):
+            if obj is BasePlatformAdapter or not issubclass(obj, BasePlatformAdapter):
+                continue
+            if obj.__module__ != module_name:
+                continue
+            name = getattr(obj, "platform_name", None) or path.stem
+            descriptor = None
+            candidate = getattr(obj, "channel_descriptor", None)
+            if candidate is not None:
+                try:
+                    descriptor = candidate() if callable(candidate) else candidate
+                except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)